### ISSUE TITLE
$d doesn't support key format object as mentionned in the docs.

### ISSUE THREAD

**Issue by existe-deja** [kazupon/vue-i18n#1502]
### Reporting a bug?

Following the [doc](https://kazupon.github.io/vue-i18n/api/#d) I expect to be able to use a `Intl.DateTimeFormat` inside the `$d` function the same way I do with `$n`. But $d only accept string key and object format isn't supported.

For exemple : `$d(new Date(), { year: 'numeric' })` But it doesn't work and I have to define all the formats in the config first.

It could be nice to support direct Intl format because you can have very specific formats you only use once.

### Expected behavior

`$d(new Date(), { year: '2-digit' })` should return `22`. Or the doc should be updated.

### Reproduction

https://codesandbox.io/s/vibrant-cartwright-hcjxc?file=/src/App.vue:161-170

### System Info

```shell
"@vue/cli-plugin-babel": "4.1.1",
"vue": "2.6.14",
"vue-i18n": "8.27.1"
```

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/kazupon/vue-i18n/blob/v8.x/CONTRIBUTING.md)
- [X] Read the [Documentation](https://kazupon.github.io/vue-i18n/)
- [X] Check that there isn't [already an issue](https://github.com/kazupon/vue-i18n/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/kazupon/vue-i18n/discussions)
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by majdkhasib** [comment]
@existe-deja What is the update on this?

**Comment by kazupon** [comment]
I've just released out v8.28.0
https://github.com/kazupon/vue-i18n/releases/tag/v8.28.0

Please check it!
